package/doc/sphinx/source/index.rst

```diff
@@ -177,6 +177,7 @@ Thank you!
    ./documentation_pages/core_modules
    ./documentation_pages/visualization_modules
    ./documentation_pages/lib_modules
+   ./documentation_pages/pickle_universe
```

Comment:
move to coordinates and link from doc/sphinx/source/documentation_pages/coordinates_modules.rst